bigframes/series.py

```diff
@@ -288,7 +289,17 @@ def __repr__(self) -> str:
         pandas_df, _, query_job = self._block.retrieve_repr_request_results(max_results)
         self._set_internal_query_job(query_job)
 
-        return repr(pandas_df.iloc[:, 0])
+        pd_series = pandas_df.iloc[:, 0]
+
+        with display_options.pandas_repr(opts):
+            import pandas.io.formats
+
+            options = pandas.io.formats.format.get_series_repr_params()  # type: ignore
```

Comment:
Ditto, let's use `kwargs` in the variable name and confirm we are working with a copy, not modifying global state. Please add a comment with what you discover regarding global state or not.

Reply:
done, added comment assuring its not global state